SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version  | Supported                   |
+| -------- | --------------------------- |
+| 0.24.x   | :white_check_mark:          |
```

Comment:
Rather than explicitly stating versions, should we instead talk about "current version" and "previous versions"?